hr_employee_spouse/README.rst

```diff
@@ -0,0 +1,14 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+======================
+Spouse for employee
+======================
+
+Module adds a field where you can specify the spouse. It also checks that two
+employees do not have the same spouse. A spouse is a partner.
+Spouse partners have a label that designates them as such.
+Security rules have been added in order to show spouse only to those able to
+see the employee they are spouse of.
+
```

Comment:
delete empty line

Reply:
... and use the oca readme template

Reply:
3afb945dc917637b41773c037c9be8fb8e48f415